manifests/profile/www_lib/php.pp

```diff
@@ -101,7 +101,8 @@
       'PHP/max_input_vars'          => '2000',
       'PHP/memory_limit'            => '256M',
       'PHP/error_reporting'         => 'E_ALL & ~E_DEPRECATED',
-      'PHP/upload_max_filesize'     => '64M',
+      'PHP/upload_max_filesize'     => '128M',
+      'PHP/post_max_size'	    => '128M',
```

Comment:
The linter is going to bark about alignment here.

Reply:
ah, yeah tab instead of spaces.  Thanks.  Fixed.